server.js

```diff
@@ -652,6 +652,30 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// Flattr integration
+camp.route(/^\/flattr\/(.*)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var thing = match[1]; // eg, `391332`
+  var format = match[2];
+  var apiUrl = 'https://api.flattr.com/rest/v2/things/' + thing;
```

Comment:
In case we can in the future, it's safer to include it in the URL. `/flattr/contributions/things/`, maybe?